examples/cmd/main.go

```diff
@@ -34,7 +35,7 @@ type DownloaderCmd struct {
 	LogHTTP bool `long:"log-http" description:"log http requests (debug level)"`
 
 	DB      string `long:"db" description:"PostgreSQL URL connection string, e.g. postgres://user:password@127.0.0.1:5432/ghsync?sslmode=disable"`
-	Token   string `long:"token" short:"t" env:"GITHUB_TOKEN" description:"GitHub personal access token" required:"true"`
+	Tokens  string `long:"tokens" short:"t" env:"GITHUB_TOKENS" description:"GitHub personal access tokens comma separated" required:"true"`
```

Comment:
maybe define it this field as `[]string`?

Reply:
gonna try 👍